nf_core/subworkflows/lint/subworkflow_tests.py

```diff
@@ -114,6 +114,22 @@ def subworkflow_tests(_, subworkflow: NFCoreComponent):
                                             snap_file,
                                         )
                                     )
+                                if "versions" in str(snap_content[test_name]) or "versions" in str(snap_content.keys()):
```

Comment:
The test I wrote for subworkflows, doesn't seem to even get to this part of the code and I'm not sure why.